lib/Controller/RoomController.php

```diff
@@ -1907,6 +1910,50 @@ public function verifyDialInPin(string $pin): DataResponse {
 		return new DataResponse($this->formatRoom($this->room, $participant, skipLastMessage: true));
 	}
 
+	/**
+	 * Direct dial-in (SIP bridge)
+	 *
+	 * @param string $phoneNumber Phone number that is called
+	 * @return DataResponse<Http::STATUS_OK, TalkRoom, array{}>|DataResponse<Http::STATUS_UNAUTHORIZED|Http::STATUS_NOT_FOUND|Http::STATUS_NOT_IMPLEMENTED, null, array{}>
+	 *
+	 * 200: Participant returned
+	 * 401: SIP request invalid
+	 * 404: Participant not found
```

Comment:
```suggestion
	 * 404: Participant not found
     * 486: Participant is currently busy
```

Reply:
I'm finding it difficult to see this behaviour without a user option?
It's like "should there be knocking" or not. And I think the answer is "it depends".
Depends on the personal preference, on the person that is calling, and the call I'm currently in.
Should it be a user option?

Reply:
486 is currently not supported :see_no_evil: 
We can add it to the openapi extractor and the Server APIs if we want, but I'm a bit unsure if it's needed as per above